api/src/httpd/router.ts

```diff
@@ -280,6 +281,21 @@ export const registerRoutes = (
       },
     );
 
+    server.get(
+      `${urlPrefix}/timestaps`,
+      silentRouteSettings(getSchema(server, "version")),
```

Comment:
@amha-mersha you have not defined the schema. You have to use correct name the schema of endpoint and also create it in `api/src/httpd/schema.ts` file and describe request and response schema there.
```suggestion
      silentRouteSettings(getSchema(server, "timestamps")),
```